libcxx/include/__functional/function.h

```diff
@@ -17,13 +17,7 @@
 #include <__functional/binary_function.h>
```

Comment:
In your commit message, can you link to the LWG issue that removed this as a DR?